CollAction/ValidationAttributes/DateWithinMonthsAttribute.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+
+namespace CollAction.ValidationAttributes
+{
+    public class DateWithinMonthsAttribute : ValidationAttribute
+    {
+        private readonly int _months;
+
+        public String OfDateProperty { get; set; }
+
+        public DateWithinMonthsAttribute(int months)
```

Comment:
I could make it more generic by passing in years, months and days as attribute parameters. I could also pass in the parameters necessary to make up a TimeSpan hours, minutes, seconds (but I think that is too fine a granularity for our purposes). Also, if we can pass in months rather than a fixed time span that gets around odd leap year behavior (off by one day).